.github/workflows/main.yml

```
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    # Limit this action to tags only
+    tags:
+      - 'v[0-9]+\.[0-9]+\.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

I think we should hardcode the version, so we know if it breaks when updating

```suggestion
    runs-on: ubuntu-20.04
```

---

Nice catch thanks. Also changed the job name from` build` to `publish`